src/services/expenses.service.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+let expenses = [];
+
+const getAllExpenses = () => {
+  return expenses;
+};
+
+const getExpenses = (params) => {
+  const userId = params.get('userId') || null;
+  const fromDate = params.get('from') || null;
+  const toDate = params.get('to') || null;
+  const categories = params.getAll('categories') || [];
+
+  if (userId) {
+    expenses = expenses.filter(
+      expense => expense.userId === +userId);
+  }
+
+  if (fromDate && toDate) {
+    expenses = expenses.filter(expense => {
+      const spentDate = new Date(expense.spentAt);
+      const from = new Date(fromDate);
+      const to = new Date(toDate);
+
+      return (spentDate >= from && spentDate <= to);
+    });
+  }
+
+  if (categories.length !== 0) {
+    expenses = expenses.filter(
+      expense => categories.some(
+        category => expense.category === category));
+  }
+
+  return expenses;
+};
+
+const getExpenseById = (id) => {
+  return expenses.find(expense => expense.id === +id) || null;
```

Comment:
```suggestion
  return expenses.find(expense => expense.id === Number(id)) || null;
```

The same in other places

Reply:
Also can avoid using `return`